examples/algorithm_progression/xarray_sweep.py

```diff
@@ -0,0 +1,67 @@
+from pathlib import Path
+
+import numpy as np
+import xarray as xr
+from algorithm_sweep import AlgorithmSweep
+
+import large_image
+
+
+class AlgorithmSweepXArray(AlgorithmSweep):
+    def initialize_storage(self):
+        self.output_files = []
+
+    def apply_algorithm(self, param_combo):
+        iteration_filename = (
+            Path(self.output_dir) / f'result_{"_".join(str(v) for v in param_combo)}.nc'
+        )
+        if not iteration_filename.exists():
+            current_coords = {
+                k: [param_combo[i]] for i, k in enumerate(self.input_params.keys())
+            }
+            source = large_image.open(self.input_filename)
+            source_metadata = source.getMetadata()
+            output_data = np.empty(
+                (
+                    source_metadata["sizeY"],
+                    source_metadata["sizeX"],
+                    4,
+                )
+            )
+            for tile in source.tileIterator(
+                format=large_image.tilesource.TILE_FORMAT_NUMPY,
+                tile_size=dict(width=2048, height=2048),
+            ):
+                output_data[
+                    tile["x"]: tile["x"] + tile["width"],
```

Comment:
Arrays default to y, x, samples not x, y, samples

Reply:
Changed in [502bcdd](https://github.com/girder/large_image/pull/1077/commits/502bcddb39506680d2ceec423f4f4b2e171dc808)